scripts/hud/cgaz.ts

```diff
@@ -821,7 +930,7 @@ class Cgaz {
 		return breakPoints;
 	}
 
-	static findSnapGains() {
+	findSnapGains() {
```

Comment:
should this explicitly return `number[]`?

Kind of related - what's the criteria for giving an explicit return type vs not?

Reply:
Good question. I'd say I'm general it's always best to add a return type, besides perhaps functions return `void` /async functions `Promise<void>`. Though when you put me on the spot and ask, I don't really see a good reason not to set a return type in that case, especially since `void` has an important *semantic* implication, i.e. that a function returning could technically return any value, but the caller must not *use* that value.

In this case, I was just being lazy and forgot to add a return type, it definitely seems helpful to be explicit in this case.

I suppose here's a cass where you might not want to specify a return type; TODO FINISH ON LAPOTP I'M NOT WRITING THIS SHIT ON PHONE KB